ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec-CUI-nonprod.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
does this file overrides the existing lr spec? perhaps discuss with Azam and Sandeep about it if it does

Reply:
Having a discussion with both and the change with be in base file as it is not impact the functionality